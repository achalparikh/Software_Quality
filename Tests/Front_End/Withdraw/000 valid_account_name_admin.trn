10
01 ACHAL PARIKH        00001 00000.00 A 
00