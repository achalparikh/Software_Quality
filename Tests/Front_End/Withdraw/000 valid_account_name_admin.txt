login
admin
withdraw
ACHAL PARIKH
logout