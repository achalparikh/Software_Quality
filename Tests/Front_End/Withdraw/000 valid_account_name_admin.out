Enter user type - "standard" or "admin":
You are now logged in as an admin
COMMAND: withdraw - enter user:
Withdrawing from "ACHAL PARIKH" - enter amount:
You are now logged out
